src/community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/writers/XHTMLTemplateWriter.java

```diff
@@ -0,0 +1,107 @@
+package org.geoserver.featurestemplating.writers;
```

Comment:
Lacks copyright header. Please check the other new files as well to make sure all have one.

Reply:
fixed.